worlds/pokemon_rb/options.py

```diff
@@ -433,6 +499,21 @@ class StartingMoney(Range):
     range_end = 999999
 
 
+class LoseMoneyOnBlackout(Toggle):
+    """Lose half your money when blacking out, as in vanilla."""
```

Comment:
```suggestion
    """Lose half your money when blacking out, like in vanilla."""
```
Not natively English speaking, but I think "like" is correct here?

Reply:
"like" sounds more informal here, to me.